src/controllers/book.js

```diff
@@ -0,0 +1,119 @@
+const db = require("../models");
+const Book = db.book;
+const Op = db.Sequelize.Op;
+
+exports.create = (req, res) => {
+  // Validate request
+  if (!req.body.title || !req.body.author) {
+    res.status(400).send({
+      message: "Title and Author are required!"
+    });
+    return;
+  }
+
+  let pubDate = new Date().toString();
+  if (req.body.publicationDate) {
+    pubDate = req.body.publicationDate.toString();
+  }
+
+  // Create a book
+  const book = {
+    title: req.body.title,
+    description: req.body.description,
+    author: req.body.author,
+    publicationDate: pubDate,
+    resume: req.body.abstract,
+    cover: req.body.cover
+  };
+
+  // Save Book in the database
+  Book.create(book)
+    .then(data => {
+      res.send(data);
+    })
+    .catch(err => {
+      res.status(500).send({
+        message:
+          err.message || "Some errors ocurred while creating the book"
+      });
+    });
+};
```

Comment:
You could try using async / await as we already talked about it, but the way you did it is great :rocket:

Reply:
yeah, I'll try that, thanks